file_paths.md

```diff
@@ -237,84 +234,122 @@ create a `puppet` user or group.
         var *
             log *
                 pxp-agent.log                 # enabled by default
-            spool *                           # directory containing results of pxp-agent modules
             run *
+            spool *                           # directory containing results of pxp-agent modules
 
-    C:\Program Files\Puppet Labs\Puppet
-        VERSION                               # puppet-agent package version
-
-    C:\Program Files\Puppet Labs\Puppet\bin *
-        environment.bat *                     # setup LOAD_PATH
-        facter.bat *                          # bat file wrappers
-        hiera.bat *
-        mco.bat *
-        puppet.bat *
-        puppet_shell.bat *                    # targets for shortcuts
-        run_facter_interactive.bat *
-        run_puppet_interactive.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\facter *
+    C:\Program Files\Puppet Labs\bin *        # symlink targets of puppet related binaries
```

Comment:
So it could be one up from `INSTALLDIR`? In this example, we assume `[INSTALLDIR] = C:\Program Files\Puppet Labs\Puppet`. Given that this is one directory level above `Puppet`, outside of `[INSTALLDIR]`, I'm wondering if this location needs to be rethought